test/basicfuns.jl

```diff
@@ -44,6 +44,37 @@
     @test iszero(xlog1py(0 + im * 0, -1 + im * Inf))
 end
 
+@testset "xexpx" begin
+    @test iszero(xexpx(-Inf))
+    @test iszero(xexpx(0))
+    @test isnan(xexpx(NaN))
+
+    @test xexpx(1) ≈ exp(1)
+    @test xexpx(2) ≈ 2exp(2)
+
+    @inferred xexpx(0)
+    @inferred xexpx(0.)
+    @inferred xexpx(0f0)
+end
+
+@testset "xexpy" begin
+    @test iszero(xexpy(Inf, -Inf))
```

Comment:
Mathematically the limit does not exist.

Reply:
Yeah but having `xexpy(Inf, -Inf) == 0` is the point of having `xexpy` at all (just like `xlogy`)

Reply:
Sure, I agree. I just think it is weird/incorrect to talk about taking limits in the docstring and one should rather explain the special case, i.e., how the implementation differs from `x * exp(-y)`.

Reply:
I updated the docstrings as suggested by @tpapp to make this point clearer.